libs/tup-components/src/system_monitor/details/SystemDetails.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { useMemo } from 'react';
+import { useTable, Column } from 'react-table';
+import { EmptyTablePlaceholder } from '../../utils';
+import {
+  LoadingSpinner,
+  SectionTableWrapper,
+  Pill,
+  Button,
+} from '@tacc/core-components';
+import { JobsQueue, useSystemQueue } from '@tacc/tup-hooks';
+import styles from './SystemDetails.module.css';
+
+const SystemQueueTable: React.FC<{
+  tas_name: string;
+}> = ({ tas_name }) => {
+  const { data: systemData, isLoading } = useSystemQueue(tas_name);
+  const system: JobsQueue[] =
+    systemData?.queues?.filter((queue) => !queue.hidden) ?? [];
+  const columns = useMemo<Column<JobsQueue>[]>(
+    () => [
+      {
+        accessor: 'name',
+        Header: 'Queue',
+      },
+      {
+        accessor: ({ down }) =>
+          down ? (
+            <Pill type="warning">Closed</Pill>
+          ) : (
+            <Pill type="normal">Open</Pill>
+          ),
+        Header: 'Status',
+      },
+      {
+        accessor: 'free',
+        Header: 'Free Nodes',
+      },
+      {
+        accessor: ({ running }) => (running ? running : ' 0 '),
+        Header: 'Running Jobs',
+      },
+      {
+        accessor: ({ waiting }) => (waiting ? waiting : ' 0 '),
+        Header: 'Waiting Jobs',
+      },
+    ],
+    []
+  );
+
+  const { getTableProps, getTableBodyProps, rows, prepareRow, headerGroups } =
+    useTable({
+      columns,
+      data: system ?? [],
+    });
+
+  if (isLoading) return <LoadingSpinner />;
+
+  return (
+    <table
+      {...getTableProps()}
+      className={`o-fixed-header-table ${styles['systems-table']}`}
+    >
+      <thead>
+        {headerGroups.map((headerGroup) => (
+          <tr {...headerGroup.getHeaderGroupProps()}>
+            {headerGroup.headers.map((column) => (
+              <th key={column.id}>{column.render('Header')}</th>
+            ))}
+          </tr>
+        ))}
+      </thead>
+      <tbody {...getTableBodyProps()}>
+        {rows.length ? (
+          rows.map((row, idx) => {
+            prepareRow(row);
+            return (
+              <tr
+                className={styles['system-listing-row']}
+                {...row.getRowProps()}
+              >
+                {row.cells.map((cell) => (
+                  <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
+                ))}
+              </tr>
+            );
+          })
+        ) : (
+          <tr>
+            <td colSpan={5}>
+              <EmptyTablePlaceholder>
+                System job queues are unavailable
+              </EmptyTablePlaceholder>
+            </td>
+          </tr>
+        )}
+      </tbody>
+    </table>
+  );
+};
+const ViewMonitor = () => (
+  <a href="/portal/system_monitor">
+    <Button type="primary">View All Systems</Button>
+  </a>
```

Comment:
I notice excess Network Activity when I click this button. The UX result is longer load time than other navigation in the Portal. I guess the cause/solution to be the use of `<a href="…">` instead of `<Link>`.

<details><summary>Video and Description</summary>

https://github.com/TACC/tup-ui/assets/62723358/36c66a4e-3d68-467e-9e7f-0e2211a3ca52

As I navigate the Portal, with the Developer Tools' "Network" tab open, filtered to only show CSS, there is no visible network activity there. This is expected, because no extra CSS is loaded. But, when I click the "View All Systems" button, I get network activity; many CSS files are loaded; network activity similar to what I would have on initially loading the Portal. This causes a flash or pause in the navigation, as white area is shown under header until assets load, so then content can load.

</details>

Reply:
I know I bothered about `<Button>` inside `<a>` on a much older PR. This is not that concern.

<sup>I'll care about that another day, when I have time to propose a solution.</sup>

Reply:
The network activity can probably be solved by replacing this `<a href=.../>`  with a `<Link to=.../>` so that the navigation happens without a full page reload.

Reply:
Updated this to use Link rather than <a>. Thanks!